### DISCUSSION TITLE
File storage path

### DISCUSSION THREAD

**Discussion by palanisamy-emis** [benzino77/clamav-rest-api#33] [Q&A]
Hi Chaps,

<img width="1005" alt="Screenshot 2022-07-07 at 4 00 35 PM" src="https://user-images.githubusercontent.com/70691178/177753365-97b2ba0d-b6c7-42e6-9b0b-6f63b26a070e.png">

-  Where can i find the Files/Path that are passed from API in docker clam av machine?
-  Will the files gets automatically deleted in docker clam av machine?

**Comment by benzino77** [comment]
CRA is not transferring files. It is passing content of the file to the clamd daemon via network stream. I have no knowledge whether clamd is writing something to disk or not when it receives network stream.
